docs/COMPATIBILITY_MATRIX.md

```diff
@@ -0,0 +1,30 @@
+| v1.0.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |                    |                    |                    |           |
+| v1.0.1                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |                    |                    |                    |           |
+| v1.0.2                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |                    |                    |                    |           |
+| v1.1.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |        :x:         |                    |                    |                    |           |
+| v1.2.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |        :x:         |                    |                    |                    |           |
+| v1.2.1                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |                    |                    |           |
+| v1.3.0                              |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |                    |           |
+| v1.3.1                              |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |                    |           |
+| v1.4.0                              |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |           |
+| v1.5.0                              |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |           |
+| v1.6.0                              |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |
+
+| Legend             | Description  |
+|:------------------:|--------------|
+| :white_check_mark: | Compatible   |
+| :warning:          | Has issues   |
+| :x:                | Incompatible |
```

Comment:
Don't think we put the legend in other matrices: https://github.com/sighupio/fury-kubernetes-networking/blob/master/docs/COMPATIBILITY_MATRIX.md


```
:white_check_mark: Compatible

:warning: Has issues

:x: Incompatible
```

Reply:
fixed